tests/cli/commands/test_addasset.py

```diff
@@ -0,0 +1,113 @@
+import os
+import shutil
+from tempfile import TemporaryDirectory
+
+import pystac
+from pystac.utils import make_absolute_href
+
+from stactools.cli.commands.addasset import create_addasset_command
+from stactools.core import move_all_assets
+from stactools.testing import CliTestCase
+from tests import test_data
+
+from .test_cases import TestCases
+
+
+def create_temp_catalog_copy(tmp_dir):
+    col = TestCases.planet_disaster()
+    col.normalize_hrefs(tmp_dir)
+    col.save(catalog_type=pystac.CatalogType.SELF_CONTAINED)
+    move_all_assets(col, copy=True)
+    col.save()
+
+    return col
+
+
+def create_temp_copy(src_path, tmp_dir, target_name):
+    temp_path = os.path.join(tmp_dir, target_name)
+    shutil.copyfile(src_path, temp_path)
+    return temp_path
+
+
+class AddAssetTest(CliTestCase):
+    def create_subcommand_functions(self):
+        return [create_addasset_command]
+
+    def test_add_asset_to_item(self):
+        with TemporaryDirectory() as tmp_dir:
+            catalog = create_temp_catalog_copy(tmp_dir)
```

Comment:
I believe the `catalog` part of this test is not necessary, since we are just testing the behavior on the `Item`. The test should be reduced to only include the components necessary to test the expected behavior.